src/ocad-to-svg.js

```diff
@@ -149,11 +149,20 @@ const patternToSvg = (colors, s) => {
 }
 
 const createSvgNode = (document, n) => {
-  const node =
-    n.text === undefined
-      ? document.createElementNS('http://www.w3.org/2000/svg', n.type)
-      : document.createTextNode(n.text)
+  let node
+  if (n.text === undefined) {
+    node = document.createElement(n.type)
+    const xmlnss = Object.entries(n.attrs || []).filter(
+      ([key, _]) => key.indexOf('xmlns') === 0
+    )
+    for (const [ns, uri] of xmlnss) {
+      node.setAttributeNS('http://www.w3.org/2000/xmlns/', ns, uri)
+    }
```

Comment:
Just checking so I understand... it looks like attributes starting with `xmlns` will be added both here as well as on line 158, but with different namespaces. Does it matter? Should we filter out `xmlns` attributes so they don't get added a second time?

Reply:
Oh, I think you're right. That's my mistake. Can you help me, what would be the appropriate workflow to correct this?

` n.attrs &&
    Object.keys(n.attrs)
      .filter(attrName => !attrName.startsWith('xmlns'))
      .forEach(attrName => node.setAttribute(attrName, n.attrs[attrName]))`

Reply:
Looks good to me!